php-console.js

```diff
@@ -181,6 +181,53 @@
                 e.preventDefault();
             });
         }
+        
+        //Save
+        if (window.localStorage) {
```

Comment:
This features are only available when localStorage is supported, but the links within the ui do not depend on this feature detect?